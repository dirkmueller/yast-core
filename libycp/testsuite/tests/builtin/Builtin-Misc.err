Parsed:
----------------------------------------------------------------------
"** checkIP **"
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
checkIP ("127.0.0.1")
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
checkIP ("277.0.3.2")
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
checkIP ("127.0.3.")
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
"** time **"
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
(time () > 1000000000)
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
"** sleep **"
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
sleep (10)
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
"** random **"
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
(srandom (10) == nil)
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
(random (10) == 5)
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
(random (10) == 6)
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
(random (10) == 5)
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
(random (10) == 1)
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
(random (10) == 8)
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
(srandom () > 1000000000)
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
"** logging **"
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
{
    // filename: "tests/builtin/Builtin-Misc.ycp"
    y2debug ("%1 is smaller than %2", 7, "13");
    y2milestone ("%1 is smaller than %2", 7, "13");
    y2warning ("%1 is smaller than %2", 7, "13");
    y2error ("%1 is smaller than %2", 7, "13");
    y2security ("%1 is smaller than %2", 7, "13");
    y2internal ("%1 is smaller than %2", 7, "13");
}
----------------------------------------------------------------------
[YCP] tests/builtin/Builtin-Misc.ycp:45 7 is smaller than 13
[YCP] tests/builtin/Builtin-Misc.ycp:46 7 is smaller than 13
[YCP] tests/builtin/Builtin-Misc.ycp:47 7 is smaller than 13
[YCP] tests/builtin/Builtin-Misc.ycp:48 7 is smaller than 13
[YCP] tests/builtin/Builtin-Misc.ycp:49 7 is smaller than 13
