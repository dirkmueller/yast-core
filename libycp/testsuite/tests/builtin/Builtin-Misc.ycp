
# ---------------------------------------------------------
#
#  Filename:	Builtin-Misc.ycp
#
#  Purpose:	test cases for all misc builtins
#
# ---------------------------------------------------------


("** checkIP **")

(checkIP ("127.0.0.1"))
(checkIP ("277.0.3.2"))
(checkIP ("127.0.3."))


("** time **")

(time () > 1000000000)


("** sleep **")

(sleep (10))


("** random **")

(srandom (10) == nil)

(random (10) == 5)
(random (10) == 6)
(random (10) == 5)
(random (10) == 1)
(random (10) == 8)

(srandom () > 1000000000)


("** logging **")

{
  y2debug ("%1 is smaller than %2", 7, "13");
  y2milestone ("%1 is smaller than %2", 7, "13");
  y2warning ("%1 is smaller than %2", 7, "13");
  y2error ("%1 is smaller than %2", 7, "13");
  y2security ("%1 is smaller than %2", 7, "13");
  y2internal ("%1 is smaller than %2", 7, "13");

}
