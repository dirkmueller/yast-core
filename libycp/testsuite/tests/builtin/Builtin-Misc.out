("** time **")
(true)
("** sleep **")
(nil)
("** random **")
(true)
(true)
(true)
(true)
(true)
(true)
(true)
("** logging **")
(nil)
